stdpopsim/slim_engine.py

```diff
@@ -1249,6 +1230,15 @@ def _recap_and_rescale(
             )
             for pop in recap_epoch.populations
         ]
+        #########################################
+        # FIX THIS BEFORE MERGING INTO MASTER!!!!
+        # NEED THE LATEST SLiM
+        #########################################
+        # TODO: remove this line once SLiM starts populating mutation times
```

Comment:
Hey @mufernando, is this still relevant? We're currently requiring slim 3.6.

Reply:
we don't need this anymore. I'll review the other parts of the PR and fix this.

Reply:
make sure we are assuming SLiM that uses mutation times first

Reply:
3.6

Reply:
fixed this